src/components/toast/toast.component.tsx

```diff
@@ -135,25 +160,24 @@ const Toast = React.forwardRef(
         >
           <ToastStyle
             isNotice={isNotice}
-            className={componentClasses}
+            className={className}
             {...tagComponent(restProps["data-component"] || "toast", restProps)}
-            {...toastProps}
+            isCenter={isCenter}
+            variant={toastVariant}
+            id={id}
+            maxWidth={maxWidth}
             ref={toastContentNodeRef}
             {...(!disableAutoFocus && {
               tabIndex,
               onBlur: () => setTabIndex(undefined),
             })}
           >
-            {!isNotice && (
-              <TypeIcon variant={toastProps.variant}>
-                <Icon type={toastProps.variant} />
+            {variant !== "notice" && (
```

Comment:
Same as before. Minor but could just use isNotice here since we've already got it.
```suggestion
            {!isNotice && (
```

Reply:
Same as before, TS errors when done as you suggested